test/jdk/sun/management/jmxremote/bootstrap/CustomLauncherTest.java

```diff
@@ -95,9 +95,7 @@ public static void main(String[] args) throws Exception {
             System.out.println("  PID           : " + serverPrc.pid());
             System.out.println("  shutdown port : " + port.get());
 
-            ProcessBuilder client = ProcessTools.createJavaProcessBuilder(
-                "-cp",
-                Utils.TEST_CLASS_PATH,
```

Comment:
Just a question: Is the `-cp` options added by the `ProcessTools.createTestJvm()`?

Reply:
It is added even by ProcessTools.createJavaProcessBuilder(..) so it is not needed here.